vcpkg/commands/update-registry.md

```diff
@@ -0,0 +1,78 @@
+---
+title: vcpkg x-update-registry
+description: Command line reference for the vcpkg x-update-registry command. Updates dowloaded content for the supplied artifact registries.
+ms.date: 05/03/2023
+---
+# vcpkg x-update-registry
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+## Synopsis
+
+```console
+vcpkg x-update-registry [--all] artifact-registry-name-or-uri...
+```
+
+## Description
+
+Downloads the registry contents of the supplied artifact registry.

```

Comment:
Add a bit of context on when the user would interact with this feature. It's not clear from this sentence why the user would run this command. What is the "problem" the user is trying to solve by running this? Imagine you're the user and you run an online search for your problem. Reading this description should clarify that they found the right page with the solution. 

Also add a "This feature does not apply to vcpkg port registries" warning.

Reply:
I'm not sure 'describe a scenario where you want to use this' is appropriate for reference level 'this is what all the switches do' documentation like this. For that, unfortunately at this time there are only embedded use cases for which I do not have great understanding at this time.

How do you want the warning worded? It already says 'artifact registry'.